app/services/data/get-arms-totals.js

```diff
@@ -0,0 +1,36 @@
+const config = require('../../../config')
+const knexConfig = require('../../../knexfile').app
+const knex = require('knex')(knexConfig)
+
+module.exports = function (omKey, teamCode) {
+  var whereObject = {
+    'assessment_staff_key': omKey,
+    'assessmentent_team_key': teamCode // typo in extract
+  }
+
+  return knex.count('sentence_type as count').select('sentence_type')
+          .from(`${config.DB_STG_SCHEMA}.arms`)
```

Comment:
How would this work in Production?

Reply:
Yea, in this case `STG` refers to the schema staging, which our ETL process loads data into, not the environment.